### PR TITLE
Add academic program entity 

### PR DESCRIPTION
**Pull request by ikoan**

### CODE REVIEW

File: lib/biola_web/versions/v2/academic_program_api.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    class AcademicProgramAPI < Grape::API
+
+      resource :academic_programs do
+        desc "Return an array of academic programs"
+        params do
+          optional :limit, type: Integer, default: 20, greater_than: 0
+          optional :offset, type: Integer, default: 0, greater_than: -1
+        end
+        get do
+          academic_programs = AcademicProgram.all
```
Comment (ikoan) [inline lib/biola_web/versions/v2/academic_program_api.rb]: No, I will fix this... thanks for the reminder

Comment (rift137) [inline lib/biola_web/versions/v2/academic_program_api.rb]: Jeff said to only serve up published academic programs.

File: lib/biola_web/versions/v2/academic_program_api.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    class AcademicProgramAPI < Grape::API
+
+      resource :academic_programs do
+        desc "Return an array of academic programs"
+        params do
+          optional :limit, type: Integer, default: 20, greater_than: 0
+          optional :offset, type: Integer, default: 0, greater_than: -1
+        end
+        get do
+          academic_programs = AcademicProgram.where('published_to_production' => 'true')
```
Comment (rift137) [inline lib/biola_web/versions/v2/academic_program_api.rb]: `academic_programs = AcademicProgram.where(published_to_production: 'true')`

Comment (rift137) [inline lib/biola_web/versions/v2/academic_program_api.rb]: Try to not use hash rockets

Comment (ikoan) [inline lib/biola_web/versions/v2/academic_program_api.rb]: Corrected syntax, thanks!

**Pull request merged**